cmd/erasure-server-pool.go

```diff
@@ -265,6 +265,8 @@ type poolAvailableSpace struct {
 	Index      int
 	Available  uint64
 	MaxUsedPct int // Used disk percentage of most filled disk, rounded down.
+
+	Valid bool
```

Comment:
"Valid" without a bit more context isn't very obvious. A comment would be great.